pkg/worker/worker.go

```diff
@@ -251,6 +250,24 @@ func NewWorker() (*Worker, error) {
 	}, nil
 }
 
+func (s *Worker) IsCRIUAvailable() bool {
```

Comment:
can we move this to the cr.go file (that's where the rest of the checkpoint restore stuff is, for the most part)

Reply:
Updated.